client/components/account-balances/balances-tab-panel.tsx

```diff
@@ -101,97 +105,109 @@ const AccountBalancesTabPanel: React.FC = () => {
-											  } )
-									}
-								/>
-							}
-						/>
-						<BalanceBlock
-							id={ `wcpay-account-balances-${ tab.currencyCode }-pending` }
-							title={ fundLabelStrings.pending }
-							amount={ tab.pendingFunds }
-							currencyCode={ tab.currencyCode }
-							isLoading={ isLoading }
-							tooltip={
-								<BalanceTooltip
-									label={ `${ fundLabelStrings.pending } tooltip` }
-									content={ interpolateComponents( {
-										mixedString: sprintf(
-											_n(
-												'The amount of funds still in the %d day pending period. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
-												'The amount of funds still in the %d day pending period. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
-												tab.delayDays,
-												'woocommerce-payments'
-											),
-											tab.delayDays
-										),
-										components: {
-											learnMoreLink: (
-												// eslint-disable-next-line jsx-a11y/anchor-has-content
-												<a
-													rel="external noopener noreferrer"
-													target="_blank"
-													href={
-														documentationUrls.depositSchedule
-													}
-												/>
+														components: {
+															learnMoreLink: (
+																// eslint-disable-next-line jsx-a11y/anchor-has-content
+																<a
+																	rel="external noopener noreferrer"
+																	target="_blank"
+																	href={
+																		documentationUrls.depositSchedule
+																	}
+																/>
+															),
+														},
+												  } )
+										}
+									/>
+								}
+							/>
+							<BalanceBlock
+								id={ `wcpay-account-balances-${ tab.currencyCode }-pending` }
+								title={ fundLabelStrings.pending }
+								amount={ tab.pendingFunds }
+								currencyCode={ tab.currencyCode }
+								isLoading={ isLoading }
+								tooltip={
+									<BalanceTooltip
+										label={ `${ fundLabelStrings.pending } tooltip` }
+										content={ interpolateComponents( {
+											mixedString: sprintf(
+												_n(
+													'The amount of funds still in the %d day pending period. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
+													'The amount of funds still in the %d day pending period. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
+													tab.delayDays,
+													'woocommerce-payments'
+												),
+												tab.delayDays
 											),
-										},
-									} ) }
+											components: {
+												learnMoreLink: (
+													// eslint-disable-next-line jsx-a11y/anchor-has-content
+													<a
+														rel="external noopener noreferrer"
+														target="_blank"
+														href={
+															documentationUrls.depositSchedule
+														}
+													/>
+												),
+											},
+										} ) }
+									/>
+								}
+							/>
+						</Flex>
+						{ tab.instantBalance && tab.instantBalance.amount > 0 && (
```

Comment:
This big diff is just wrapping the above `<Flex>` with `<></>` and adding a conditional instant deposit button.